### ISSUE TITLE
PackageManager error on build

### ISSUE THREAD

**Issue by scriptjs** [vmware/photon#428]
Hello. I have set up latest ubuntu 14.04.4 for a build from master. I am not running vagrant for this.  I am launching the iso-minimal build iwith the following:

```
sudo make iso-minimal
```

The packages appear to build, however the following is being thrown.
```
ts/bitbucket.org/devscript/photon/stage/photonroot/build-rpm
2016-03-26 00:42:55,210 - PackageManager - INFO - Waiting to acquire scheduler lock
2016-03-26 00:42:55,211 - PackageManager - INFO - Checking for next possible packages to build
2016-03-26 00:42:55,211 - PackageManager - INFO - Waiting for all remaining worker threads
2016-03-26 00:42:55,211 - PackageManager - INFO - Released scheduler lock
2016-03-26 00:42:55,212 - PackageManager - INFO - Thread WorkerThread0 is going to rest
2016-03-26 00:42:55,212 - PackageManager - INFO - All packages built successfully
2016-03-26 00:42:55,212 - PackageManager - INFO - Terminated
2016-03-26 00:42:55,214 - PackageManager - INFO - List of Already built packages
2016-03-26 00:42:55,214 - PackageManager - INFO - ['linux-api-headers', 'perl-File-Which', 'man-pages', 'perl-DBIx-Simple', 'perl-Object-Accessor', 'perl-File-HomeDir', 'nspr', 'nss', 'filesystem', 'mpfr-devel', 'util-linux-lang', 'libgomp-devel', 'openssl-c_rehash', 'flex-devel', 'rpm-build', 'grep', 'libgcc-atomic', 'libgcc', 'expat', 'bzip2', 'perl', 'util-linux', 'patch', 'bison', 'binutils', 'elfutils-libelf-devel-static', 'libffi', 'gmp', 'python2', 'elfutils-devel', 'python2-devel', 'libgcc-devel', 'perl-DBI', 'file', 'm4', 'elfutils-libelf', 'coreutils-lang', 'gettext', 'python2-libs', 'procps-ng-devel', 'readline-devel', 'nss-devel', 'lua', 'libltdl', 'bzip2-devel', 'findutils', 'libtool', 'cpio', 'coreutils', 'python2-tools', 'util-linux-devel', 'sed', 'popt-devel', 'groff', 'flex', 'xz-devel', 'gawk', 'libstdc++-devel', 'make', 'rpm', 'glibc', 'libgomp', 'autoconf', 'mpc', 'zlib-devel', 'rpm-devel', 'sed-lang', 'elfutils', 'pkg-config', 'procps-ng', 'bash', 'python-curses', 'ncurses', 'zlib', 'gdbm', 'glibc-devel', 'readline', 'glibc-lang', 'popt', 'binutils-devel', 'texinfo', 'man-db', 'python-xml', 'libstdc++', 'perl-DBD-SQLite', 'diffutils', 'openssl', 'lua-devel', 'tar', 'mpfr', 'libltdl-devel', 'elfutils-libelf-devel', 'elfutils-devel-static', 'gzip', 'openssl-perl', 'openssl-devel', 'sqlite-autoconf', 'gmp-devel', 'ncurses-devel', 'xz', 'automake', 'bash-lang', 'libpipeline', 'gcc']
2016-03-26 00:42:55,214 - PackageManager - INFO - Reading dependency graph to check for cycles
2016-03-26 00:42:55,215 - Serializable Spec objects - ERROR - Could not able to find iso-minimal package from specs
2016-03-26 00:42:55,215 - PackageManager - ERROR - unable to get sorted list
2016-03-26 00:42:55,215 - PackageManager - ERROR - Unable to set paramaters. Terminating the package manager.
```

I don't wish to mix another issue with the previous but I am receiving the warnings messages in the build that may be relevant, ie
```
warning: %post(bash-4.3.30-2.ph1.x86_64) scriptlet failed, exit status 127
/bin/sh: error while loading shared libraries: libreadline.so.6: cannot open shared object file: No such file or directory
```
Can you explain these. I will look into previous issues in the interim to see if I can located anything more about this and attempt to see what is up with the PackageManager.

**Comment by scriptjs** [comment]
An update to this.  I have built the full iso that will prompt for the an install type during installation where minimal is a choice with docker 1.10.2 running. Would still appreciate feedback on the above warnings. Am hoping the when minimal installed the build will be just as minimal. The full build requires some hours building > 800 rpms. Many thanks.

**Comment by YustasSwamp** [comment]
Hi,

warning: %post(bash-4.3.30-2.ph1.x86_64) scriptlet failed, exit status 127
/bin/sh: error while loading shared libraries: libreadline.so.6: cannot
open shared object file: No such file or


That is expected warning/error.

Thanks,
--Alexey

**Comment by suezzelur** [comment]
@scriptjs Did you try sudo make minimal-iso for making the minimal iso? iso-minimal is not a valid target.

**Comment by scriptjs** [comment]
@seuzzelur Hi. That did not work for me. I used:

```
sudo make iso
```

This makes an ISO where you can choose minimal install during installation. Thus there is a bug for the target in the makefile where the iso-minimal should be fixed or removed.

**Comment by suezzelur** [comment]
Are you referring to this target in makefile? https://github.com/vmware/photon/blob/master/Makefile#L99

**Comment by scriptjs** [comment]
Hi. My preference is for a minimal only build.  Yes that is what I am referring to.  At this point the docker included is also behind on version and needs to be upgraded. I will file an issue there since there were several bugs fixes since 1.10.2. I have started my own package repo. Because docker is the primary reason for using photon, I am hoping you will be keeping it up to date with current releases. 1.10.2 was a Feb release and we are now in April and there was no 1.10.3 available in your repo. 1.11.0 was released yesterday from what I am seeing.
